lib/commands/new/loaders/require.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const ProjectItem = require('../../../project-item').ProjectItem;
+
+module.exports = function(project) {
+  project.loader = project.model.loader.id;
+  project.model.loader = undefined; // remove loader from model as it is actually a property of model.build 
```

Comment:
Do we need that property on model then? Is it just noise that isn't actually used?